playbooks/roles/node-exporter/meta/main.yml

```diff
@@ -16,3 +16,4 @@ dependencies:
     NGINX_PROXY_PUBLIC_PORT: 19100
     NGINX_PROXY_INTERNAL_PORT: 9100
     NGINX_PROXY_USE_SSL: "{{ NODE_EXPORTER_USE_SSL }}"
+    NGINX_PROXY_EXTRA_CONFIG: ""
```

Comment:
Why did this need to be added?

Reply:
To set a default value because if the `nginx-proxy` role is used at another moment in the playbook, it may use the already defined value. Which is the case for the `jira` role.

Related discussion: https://chat.opencraft.com/opencraft/pl/4hfzcctu9intxearsiy5m9akfo

Reply:
Ah wow :grimacing: nice catch. :+1: